openapi/nexus.json

```diff
@@ -7,7 +7,7 @@
       "url": "https://oxide.computer",
       "email": "api@oxide.computer"
     },
-    "version": "20241204.0"
+    "version": "20241204.0.0"
```

Comment:
I hadn't considered this consequence. I can't see any problems with it; this version string does appear in CLI/SDK stuff...